lib/tasks/checkins.rake

```diff
@@ -48,4 +48,16 @@ namespace :checkins do
       end
     end
   end
+
+  desc 'Imports checkins from CSV'
+  task :import, [:filepath, :device_id] => :environment do |_t, args|
+    Checkin.transaction do
+      CSV.foreach(args[:filepath], headers: true) do |row|
+        checkin = Checkin.find_by_id(row['id']) || Checkin.new
```

Comment:
@dmitrytrager I'm not sure a bulk import will be possible here as we want to be able to update existing check-ins OR create new ones depending on whether the checkins being imported are already in the database.